supported_formats_and_operations.md

```diff
@@ -61,6 +67,7 @@ Which consumer supports which preprocessing operation?
 |  `clip`                    | ?       | ?          | ?    | 
 |  `min_max`                 | ?       | ?          | ?    |
```

Comment:
We should rename this to scale_min_max to be consistent with the post-processing and scale_linear.

Could you also check if we have the same definition for scale_min_max for pre- and post- processing? That way, we can simplify the implementation by sharing them.


BTW, do we really need to group them into pre- and post- processing for the implementation? Some functions might not make sense to be used in both, but some does. It might be easier to describe it as a pool of functions that one can used both for pre- and post-processing.

Reply:
> We should rename this to scale_min_max to be consistent with the post-processing and scale_linear.

I agree,

> BTW, do we really need to group them into pre- and post- processing for the implementation? Some functions might not make sense to be used in both, but some does. It might be easier to describe it as a pool of functions that one can used both for pre- and post-processing.

Yes, I have also been thinking about this. The only thing that is specific to post-processing right now is the `reference_input`. But I agree that it would make it easier to just maintain a single list and then just write a comment for functions that are not suitable for pre/post-procesing.

I will change it later.

Reply:
Or we drop `scale_min_max` and leave `percentile` and `scale_linear`. This might help some backends to implement only one while complying with the config and keeps functionality nicely separated. What do you think?

Reply:
I also looked at this again and I am not so sure what to do here. Right now combining everything into just one category doesn't really work because of different function signatures in pre/post-processing, e.g. for `scale_min_max`.

Reply:
I started https://github.com/bioimage-io/configuration/pull/61 to decouple this discussion from the rest of the changes in this PR. There I attempted to unify pre and postprocessing more..